persistence/src/test/java/com/epam/rft/atsy/persistence/repositories/CandidateRepositoryIT.java

```diff
@@ -336,4 +337,26 @@ public void findByEmailShouldFindExistingCandidate() {
     assertThat(candidate, notNullValue());
     assertThat(candidate.getEmail(), is(EXISTING_CANDIDATE_EMAIL));
   }
+
+  @Test
+  public void findByCandidateFiletRequestShouldFindCandidateAOnlyByPhoneFilter() {
+    //Given
+    String name = "";
+    String email = "";
+    String phone = CANDIDATE_A_PHONE;
+    String position = "";
```

Comment:
Minor improvement suggestion: these local variables (name, email, phone, position) can be final.